README.md

```diff
@@ -379,6 +379,24 @@ git clone https://github.com/alexesba/nvim-config.git
 
 </h3>
 
+### Optional Installation
+
+<h3>
```

Comment:
Can you please change the <h3> to regular markdown syntax? 

I'm using the h3 only for the installation process. but for the other elements, I would prefer to have a simple markdown :P

Reply:
done!